build-system/check-package-manager.js

```diff
@@ -154,6 +154,7 @@ function getYarnStableVersion(infoJson) {
 }
 
 function checkGlobalGulp() {
+  const firstInstall = getStdout('ls').trim().match(/node_modules/) === null;
```

Comment:
A cleaner way to do this would be to use `existsSync` from the `fs-extra` library.

See this file for an example:

https://github.com/ampproject/amphtml/blob/28aae733d08a62a06003e1a390452c026c61a955/build-system/tasks/update-packages.js#L19

Reply:
Ahh ok. Was trying to avoid to import new things. Thanks!

Reply:
As @Marcial1234 found out, we can't import any packages in `check-package-manager.js` since it runs before the `node_modules` directory is populated. We'd need to make this check work with native nodejs and be platform independent.

Reply:
Fixed. Is the note at the top of the file sufficient/comprehensible?